mxabierto/adela
# update Gemfile.lock from rubyzip (1.2.0) to rubyzip (1.2.1)

### Changelog

* TODO: Updated library in File Gemfile.lock from rubyzip from version 1.2.0 to rubyzip 1.2.1 in order to avoid vulnerability.

### How to test

1)  Download this source code.
2)  type in the console bundle install.
3) You can see the library rubyzip now is the version 1.2.1 instead of 1.2.0.
4) At the moment of aprove this change in github the vulnerability has dessapear.
